Kconfig.zephyr

```diff
@@ -428,6 +428,12 @@ config NO_OPTIMIZATIONS
 	  default stack sizes in order to avoid stack overflows.
 endchoice
 
+config LTO
+	bool "Link Time Optimisation"
```

Comment:
Minor nit.

Zephyr uses the optimization` spelling form here and other places:
https://github.com/zephyrproject-rtos/zephyr/blob/7939cb83dc8ed8700b3ee6ca95fba08d420a977c/Kconfig.zephyr#L394

Reply:
Done